Jumbleberry/Cookbooksv2
# 20.04

Implements/Fixes compatibilities between recipes to enable:
 1. Fully working `configure::base` on Ubuntu 18.04, 20.04 (does not work on 16.04 due to deleted packages from apt)
 2. Fully working `configure` on Ubuntu 16.04, 18.04, 20.04

To test, please try to provision boxes via:
```
AWS_PROFILE=production packer build -var 'lsb_release=18.04' -var 'lsb_codename=bionic' -only=base -on-error=ask boxes.json 
```
and 
```
AWS_PROFILE=production packer build -var 'lsb_release=20.04' -var 'lsb_codename=focal' -only=base -on-error=ask boxes.json 
```

Lastly, `vagrant up` on the dev box should work for all base builds: 16.04, 18.04, 20.04. 
The box builder does not natively support versioning these accordingly, so I would recommend testing in order:

1. vagrant up (default 16.04 box)
2. packer build on 20.04
3. vagrant destroy dev -f && vagrant up dev (post packer build will be using a 20.04 base)
4. packer build 18.04
5. vagrant destroy dev -f && vagrant up dev (post packer build will be using a 18.04 base)

We can use 18.04 for standardization until OpsWorks comes around supporting 20.04.
